src/win32/posix_w32.c

```diff
@@ -296,7 +312,23 @@ int p_open(const char *path, int flags, ...)
 		va_end(arg_list);
 	}
 
-	return _wopen(buf, flags | STANDARD_OPEN_FLAGS, mode & WIN32_MODE_MASK);
+	/* wait up to 50ms if file is locked by another thread or process */
+	open_tries = 0;
+	while (open_tries < 10) {
+		handle = _wopen(buf, flags | STANDARD_OPEN_FLAGS, mode & WIN32_MODE_MASK);
+		if (handle != -1) {
+			break;
+		}
+
+		if (errno == EACCES) {
```

Comment:
My interpretation of the docs here: https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx was that it could be `EACCES`, `EEXIST`, `EINVAL`, `EMFILE`, or `ENOENT`. `EACCES` was the only one I wanted to retry on, but if you'd like I can kill the conditional and we can just retry no matter the errno.